Core/HW/StereoResampler.h

```diff
@@ -25,6 +25,7 @@
 
 #include "Common/ChunkFile.h"
 #include "Common/CommonTypes.h"
+#include "Core/WaveFile.h"
```

Comment:
I believe that was leftover from a previous attempt at dumping audio, will move/remove.